### PR TITLE
Text improvements landing page

### LINKED ISSUE #368: exchange text for mobile version
The "Ich brauche Hilfe" and "Ich möchte helfen" buttons need to be repositioned so that they show up above the corresponding textbox (mobile version)

### PR DESCRIPTION
**Pull request by mauriceackel**
**What changes does this PR introduce**

We now only show a shortened (combined) version of the text on the landing page instead of the text of the two columns below each other.

Closes #368

### CODE REVIEW

**Review by florianschmidt1994 — APPROVED** [review]
I trust you on this one, since it's only text changes anyways :)

File: public/locales/de/translation.json [hunk 1]
```diff
@@ -52,8 +52,9 @@
...
+        "needHelp": "Ihr seid in Quarantäne oder solltet gerade besser nicht das Haus verlassen, weil Ihr zu einer <strong>Risikogruppe</strong> gehört? Seid <strong>Held*innen</strong> und <strong>bleibt zuhause</strong>, um die Verbreitung von Corona aktiv zu verlangsamen. Schreibt einfach eine Hilfsanfrage und lasst Euch zum Beispiel bei <strong>Einkäufen</strong>, <strong>Botengängen</strong> oder <strong>Gassigehen</strong> unterstützen.",
+        "combined": "Aktuell tragen alle ihren Teil zur <strong>Eindämmung</strong> der Pandemie bei, indem sie ihre Kontakte so gut es geht einschränken. Wer in Quarantäne ist, zu einer Risikogruppe gehört oder vielleicht selbst eine Ansteckungsgefahr darstellen könnte ist jedoch auf <strong>Unterstützung</strong> von außerhalb angewiesen und nicht jede*r hat Menschen in der Nähe, die einem helfen können. Über unsere Plattform vermitteln wir Hilfesuchende an Helfende - sodass Hilfesuchende Unterstützung für <strong>Einkäufen</strong>, <strong>Botengängen</strong> oder <strong>Gassigehen</strong> finden können."
```
Comment (jagaSto) [inline public/locales/de/translation.json:57]: this key is not used anywhere no?

File: src/views/Main.jsx [hunk 2]
```diff
@@ -130,6 +130,12 @@ export default function Main() {
             <Trans i18nKey="views.main.explanations.wantToHelp" />
           </p>
         </div>
+
+        <div className="md:hidden p-4 font-open-sans flex flex-col md:flex-row space-between md:p-0 mb-4 md:mb-8">
+          <p className="hyphens-auto md:leading-7 text-justify flex-1">
+            <Trans i18nKey="views.main.explanations.needHelp" />
```
Comment (jagaSto) [inline src/views/Main.jsx:136]: I guess this one should use the `combined` key :)

Screenshot 2021-01-01 at 14 26 36

File: public/locales/de/translation.json (hunk 1 above)
Comment (jagaSto) [inline public/locales/de/translation.json:57]: `Über unsere Plattform vermitteln wir Hilfesuchende an Helfende - sodass Hilfesuchende Unterstützung...` --> `Über unsere Plattform vermitteln wir Hilfesuchende an Helfende - sodass diese Unterstützung...` ?

**Comment by mauriceackel** [comment]
@jagaSto Should be fixed, can you check again and merge?

**Review by jagaSto — APPROVED** [review]
👍

**Pull request merged**